<launch>

	<node pkg="MOOS" type="MOOSDB" name="MOOSDB" />
    
    <node pkg="MOOS" type="uPlayBack" name="uPlayBack" output="screen" />
    
    <node pkg="fhwa2_MOOS_to_ROS" type="mapBridge.py" name="moos2rviz_survey"
        output="screen" args="$(find fhwa2_MOOS_to_ROS)/cfg/survey.yaml">
        <param name="prefix" value="$(find fhwa2_MOOS_to_ROS)" />
    </node>

    <node pkg="fhwa2_MOOS_to_ROS" type="liveBridge_node" name="fhwa2_testcpp"
        output="screen" respawn="true" />

    <node pkg="rviz" type="rviz" name="rViz"
        args="-d $(find fhwa2_MOOS_to_ROS)/launch/fhwa2.vcg"
        required="true" output="screen" />


</launch>